src/lambda/evaluation_context.rs

```diff
@@ -49,6 +49,10 @@ impl<'a, Ctx: ResolverContextLike<'a>> EvaluationContext<'a, Ctx> {
     value.to_str().ok()
   }
 
+  pub fn env_var(&self, key: &str) -> Option<String> {
```

Comment:
```suggestion
  pub fn env_var<'a>(&'a self, key: &'a str) -> Option<&'a str> {
```

Can we do this?

Reply:
![image](https://github.com/tailcallhq/tailcall/assets/100464352/602ffb8a-5fbc-466e-8f76-0c132fd2edc4)

I see the error when implementing the lifetimes, but I changed the return to `&str` and it worked. I've committed it.